packages/client/tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "skipLibCheck": true,
     "strict": true,
     "tsBuildInfoFile": "node_modules/.cache/tsbuildinfo.json",
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "lib": ["esnext"]
```

Comment:
`lib/dom` was overriding the types so we need to explicitly define the lib here. Main reason was that `Headers.entries()` wasn't typed.